content/rs/administering/cluster-operations/updating-certificates.md

```diff
@@ -22,7 +22,35 @@ on the browser you use, you might be able to allow the connection for
 this specific session, or add an exception to make this site trusted in
 future sessions.{{% /note %}}
 
-## How to update SSL/TLS certificates
+## How to update TLS certificates in RS 5.4.x and Later
+
+You can replace an existing certificate on the cluster leveraging the rladmin command-line utility.
+
+There are 5 different types of certificates that can be replaced starting in RS version 5.4.x. These may be replaced leveraging the below command.
+
+- API - This is the certificate used by the API
+- CM  - This is the certificate used by the management user interface or the cluster manager. It is also used for the sentinel discovery service if you are using sentinel
+- Proxy  - This is the certificate used to establish communication between a client and the databases
+- Syncer  - This is the certificate used to encrypt replication between clusters 
+- Metrics_Exporter - This is the certificate used by Prometheus.
+
+The below command syntax can be used to replace certificates substituting the below variables:
+
+- certificate-type - The type of certificate you want to replace
+- key-file-name - The name of your non-password protected key file.
+- certificate-file-name - The name of your certificate file
+
+    ```bash
+ rladmin cluster certificate set <certificate-type> certificate_file <certificate-file-name>.pem key_file <key-file-name>.pem
+    ```
+
+For example, the following command would replace the cm certificate with the private key "key.pem" and the certificate file "cluster.pem"
+   
+   ```bash
+rladmin cluster certificate set cm certificate_file cluster.pem key_file key.pem
+   ```
+
+## How to update SSL/TLS certificates in earlier version of RS
```

Comment:
@IAmATeaPot418 We have an article for earlier versions at https://docs.redislabs.com/latest/rs/references/procedures-previous-releases/. Can we put this information there?

Reply:
Yes - agree there completely! I'll move this one over.